test/rubygems/test_gem_stream_ui.rb

```diff
@@ -4,9 +4,9 @@
 require "timeout"
 
 class TestGemStreamUI < Gem::TestCase
-  # increase timeout with MJIT for --jit-wait testing
-  mjit_enabled = defined?(RubyVM::MJIT) && RubyVM::MJIT.enabled?
-  SHORT_TIMEOUT = RUBY_ENGINE == "ruby" && !mjit_enabled ? 0.1 : 1.0
+  # increase timeout with RJIT for --jit-wait testing
+  rjit_enabled = defined?(RubyVM::RJIT) && RubyVM::RJIT.enabled?
+  SHORT_TIMEOUT = RUBY_ENGINE == "ruby" && !rjit_enabled ? 0.1 : 1.0
```

Comment:
As far as rubygems supports ruby 3.2 or earlier, isn't it needed to check `RubyVM::MJIT.enabled?` too?